client/consensus/manual-seal/src/consensus/babe.rs

```diff
@@ -150,6 +186,43 @@ impl<B, C> ConsensusDataProvider<B> for BabeConsensusDataProvider<B, C>
 			.map_err(|e| Error::StringError(format!("failed to fetch epoch data: {}", e)))?
 			.ok_or_else(|| sp_consensus::Error::InvalidAuthoritiesSet)?;
 
+		let epoch = epoch_changes
```

Comment:
This (+ the context) seems like a duplicate of what is in `create_digest`, any way to DRY it?